internal/strict/strict.go

```diff
@@ -103,6 +106,10 @@ var StrictControls = Controls{
 		Error:   errors.New("The `validate-all` command is no longer supported. Use `terragrunt run-all validate` instead."),
 		Warning: "The `validate-all` command is deprecated and will be removed in a future version. Use `terragrunt run-all validate` instead.",
 	},
+	SkipDependenciesInputs: {
+		Error:   errors.New(fmt.Sprintf("The `%s` option is deprecated. Reading inputs from dependencies has been deprecated and will be removed in a future version of Terragrunt. To continue using inputs from dependencies, forward them as outputs.", SkipDependenciesInputs)),
```

Comment:
I just noticed that this package uses golang's native `errors` package, which means that when an error is created, no stack trace is included, this is also why you didn't find the `errors.Errorf` function.

Would you consider replacing it?

```go
import "github.com/gruntwork-io/terragrunt/internal/errors"
```

Reply:
Moved to `github.com/gruntwork-io/terragrunt/internal/errors`